src/xhr.js

```diff
@@ -71,18 +128,18 @@ class Xhr {
  * us to immediately support a much wide API.
  *
  * @param {string} input
- * @param {?FetchInit=} opt_init
+ * @param {!FetchInit=} init
  * @return {!Promise<!FetchResponse>}
  * @private Visible for testing
  */
-export function fetchPolyfill(input, opt_init) {
+export function fetchPolyfill(input, init) {
   assert(typeof input == 'string', 'Only URL supported: %s', input);
-  const init = opt_init || {};
+
```

Comment:
But, you still have to check/default it here :) This is an independent API and just as the global `fetch`, this method allows an optional `opt_init`. The fact that `fetchJson` will always send defined value here is fully `fetchJson`'s business.